librz/util/table.c

```diff
@@ -60,16 +61,14 @@ static void __table_adjust(RzTable *t) {
 	}
 }
 
-RZ_API void rz_table_row_free(void *_row) {
+RZ_API void rz_table_row_free(void *_row, void *user) {
```

Comment:
what is the purpose of the new `user` pointer?

Reply:
Please add Doxygen also

Reply:
Please also handle NULL